packages/services/schema/src/lib/compose.ts

```diff
@@ -46,11 +45,15 @@ const EXTERNAL_COMPOSITION_RESULT = z.union([
         }),
       ),
     }),
-    includesNetworkError: z.boolean().default(false),
```

Comment:
We never want to receive this from the external composition service. It i state that only exists on our end.